### DISCUSSION TITLE
Can I install the application on the window OS without admin permission ?

### DISCUSSION THREAD

**Discussion by bobha92** [tw93/Pake#797] [Q&A]
Can I install the application on the window OS without admin permission? Since I really want to use these useful app but it always required the admin permission to install it, is there any alternative way to bypass this? Appreciate your reply in advance!

**Comment by philippe-granet** [comment]
Same question

**Comment by o-l-a-v** [comment]
Maybe lssmsi can be used tobextract the MSI instead? That's what Scoop (Windows package manager) does with MSIs.

<https://lessmsi.activescott.com/>

Some MSI installers also support installing as current user with a command line argument.
